tests/FakeItEasy.Tests/CompletedFakeObjectCallExtensionsTests.cs

```diff
@@ -1,62 +0,0 @@
-namespace FakeItEasy.Tests
-{
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Linq.Expressions;
-    using FakeItEasy.Core;
-    using FakeItEasy.Expressions;
-    using FakeItEasy.Tests.TestHelpers;
-    using FluentAssertions;
-    using Xunit;
-
-    public class CompletedFakeObjectCallExtensionsTests : ConfigurableServiceLocatorTestBase

```

Comment:
Why remove these tests? Couldn't they be updated to reflect the new signature of `CreateCallMatcher`?